posthog/models/activity_logging/activity_log.py

```diff
@@ -237,54 +239,57 @@ def changes_between(
     previous: Optional[models.Model],
     current: Optional[models.Model],
 ) -> list[Change]:
-    """
-    Identifies changes between two models by comparing fields
-    """
     changes: list[Change] = []
 
     if previous is None and current is None:
-        # there are no changes between two things that don't exist
         return changes
 
-    if previous is not None:
-        fields = current._meta.get_fields() if current is not None else []
+    if previous is not None and current is not None:
+        fields = current._meta.get_fields()
```

Comment:
i don't think this is the same
previously we'd have run this branch when there was a previous, now we only run it if there's a previous and a current
it's a while since i looked here though - do we know we always have both?
e.g. if we delete something is current then None?

Reply:
ooops, that was an oversight from me end.